code/game/objects/items/weapons/blades.dm

```diff
@@ -196,6 +203,20 @@
 	loaded_reagent = null
 	return ..()
 
+/obj/item/weapon/claymore/harvester/spear
+	name = "\improper HP-S Harvester spear"
+	desc = "TerraGov Marine Corps' experimental High Point-Singularity 'Harvester' spear. An advanced weapon that trades sheer force for the ability to apply a variety of debilitating effects when loaded with certain reagents. Activate after loading to prime a single use of an effect. It also harvests substances from alien lifeforms it strikes when connected to the Vali system."
+	icon_state = "vali_spear"
+	item_state = "vali_spear"
+	force = 32
+	flags_equip_slot = ITEM_SLOT_BACK
+	force_wielded = 60
```

Comment:
Its parent already declares `force_wielded` as 60, so you wouldnt need it here.

Reply:
removed

Reply:
But wouldn't they actually need it here? The force_wielded of the harvester sword is 40, not 60.

Reply:
since spear is onehanded now, it doesnt matter